contractdata/AMBROSE/_AMBROSE_MASTERY.json

```diff
@@ -0,0 +1,85 @@
+{
+  "Id": "LOCATION_PARENT_ROCKY",
```

Comment:
General comment about the mastery data:
If you require max level, you can add it under here e.g.
```json
{
    "Id": "LOCATION_PARENT_ROCKY",
    "MaxLevel": 20,
    "Drops": [...]
}
```

Reply:
I determined the MaxLevel based on the highest mastery drop, so not needed. They all match up. Only exception would be Evergreen, where there is a Mastery up til 25, but we could technically make it go indefinately (especially since the Safehouse has more unlocks in it post level-25).

Reply:
Adding MaxLevel as a property would reduce the time seeking for the max level, having it default to 20 and an optional property sounds good. Only exceptions are Evergreen and Carpathian.